site/src/sphinx/server-sse.rst

```diff
@@ -49,3 +53,37 @@ use the annotation.
     public Publisher<ServerSentEvent> sse() {
         return Flux.just(ServerSentEvent.ofData("foo"), ServerSentEvent.ofData("bar"));
     }
+
+Adjusting the request timeout
+-----------------------------
+
+The events may being sent for a longer time depending on the application. Even it can continue infinitely,
+for example streaming stock quotes. But the server allows receiving a request and sending the corresponding
+response completely only within the amount of time, which is configured as the request timeout property.
+Even if you do not configure the property, it is configured to ``10`` seconds by default.
+So you must adjust the timeout if your event stream is possible to exceed it because the response will be
```

Comment:
- So -> Therefore,
- if your event stream lasts longer than the configured timeout.
- because the response will be ... -> Could be removed, because we explained it above already?